src/modules/mana/sagas.spec.data.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I had my doubts with this and commented it before on the PR. Previously thought about the __test__ folder, but that goes against the file structure standards defined in https://github.com/decentraland/standards/blob/master/standards/react-redux.md#1-directory-structure.

I separated the data from the tests because having those long inputs in the test file seemed to be cumbersome.

Reply:
Moved all data into the test file directly